examples/csv-export/index.js

```diff
@@ -62,6 +62,7 @@ class Example extends React.Component {
           filename: 'excel-format.csv',
           separator: ';',
       },
+      onDownload: () => false,
```

Comment:
So, the only issue with this is that it defaults to off for people testing csv export functionality in the example. I was thinking something like this might be more instructive:

```
state = { isOnDownload: true };

...

const options = { onDownload: () => this.state.isOnDownload };

...

<button onClick={ () => this.setState({ isOnDownload: false }) }>Shut off download functionality</button>
```

Does that make sense?

Reply:
Yep, makes sense. Updated and pushed!

Reply:
Thank you!